packages/apps/src/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
```

Comment:
Does it mean for every component.. we will define file which would contain the css changes? meaning it will over write the default styled component with the css property we define?

Reply:
I am not sure if I correctly get your concern, but I will try to explain:

- We remain defining css styles like what Polkadot is doing in each component;

- We defined color shades in a structure, so that we do not use color code in components directly, instead we use colors like `colors.N900` or `colors.primary` or `colors.success`, so that whenever design team want to redefine colors, we just need to change them in one place [here](https://github.com/cennznet/apps/pull/11/files#diff-651ef4fddd0f265387d797bee86bd8bbR1)

- We only use `defaultThemeStyle` and `computedThemeStyle`, when we want to get styles be easily configured for different theming, for example, here we defined `background: colors.N900`, which means later if we want to quickly retheming cennznet-ui to be like plug-ui (where SideBar background is pink for example), we do not need to change code here, instead we just need to define the pink color in [components.ts](https://github.com/cennznet/apps/pull/11/files#diff-d34099a3f9e037737bf239ef691e521cR6)